src/olympia/addons/serializers.py

```diff
@@ -483,6 +484,7 @@ def fake_object(self, data):
 
         obj.average_rating = data.get('ratings', {}).get('average')
         obj.total_reviews = data.get('ratings', {}).get('count')
+        obj.text_reviews = data.get('ratings', {}).get('text_count')
```

Comment:
Since there are now 3 lines with the same pattern... Maybe at least move `data.get('ratings', {})` to its own variable?